com.ktechpit.colorwall.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: com.ktechpit.colorwall
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: colorwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --device=dri
+  - --env=PACKAGE_TYPE=flatpak
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-run/dconf
```

Comment:
you do not need dconf permission. They have been built into the runtime for a few years now.

Reply:
Removed. 
Thanks